.infrastructure/helm-chart/todoapp/templates/rbac.yml

```diff
@@ -0,0 +1,30 @@
+kind: ServiceAccount
+apiVersion: v1
+metadata:
+  name: {{ .Values.todoapp.RBAC.name }}
+  namespace: {{ .Values.todoapp.namespace }}
```

Comment:
Ensure that the `todoapp.namespace` value is defined in your `values.yaml` file. If it is not set, this could lead to errors during deployment as the namespace would be undefined.